Author: Andreas Beckmann <anbe@debian.org>
Description: allow overriding BASE_LLVM_VERSION, e.g. for minor llvm releases

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.3)
 
+if(NOT BASE_LLVM_VERSION)
 set (BASE_LLVM_VERSION 11.0.0)
+endif(NOT BASE_LLVM_VERSION)
 set(LLVM_SPIRV_VERSION ${BASE_LLVM_VERSION}.0)
 
 option(LLVM_SPIRV_INCLUDE_TESTS
